models/endoscopic_inbody_classification/configs/multi_gpu_train.json

```diff
@@ -34,6 +34,8 @@
         "$@train#trainer.run()"
     ],
     "finalize": [
-        "$dist.is_initialized() and dist.destroy_process_group()"
+        "$dist.is_initialized() and dist.destroy_process_group()",
+        "$@train_close",
+        "$@val_close"
```

Comment:
Thanks @Nic-Ma . Without these changes, the unit test should raise an error but I checked and found that the error return code is not be used on the unit test.
I made the change in:
https://github.com/Project-MONAI/model-zoo/pull/446/files#diff-6414ae8376d65a790be1ceccc60606e0d5dacdf30b283342f88e83c355720c13R57

and now it can raise errors normally.